modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/form_instance_records_search_container.jsp

```diff
@@ -125,8 +125,9 @@ PortletURL portletURL = ddmFormViewFormInstanceRecordsDisplayContext.getPortletU
 						</c:when>
 						<c:when test="<%= StringUtil.equals(ddmFormField.getType(), DDMFormFieldTypeConstants.CHECKBOX) %>">
 							<c:choose>
-								<c:when test='<%= StringUtil.equals(ddmFormViewFormInstanceRecordsDisplayContext.getColumnValue(ddmFormFieldsMap.get(ddmFormField.getName()), StringPool.BLANK, ddmFormFieldValuesMap.get(ddmFormField.getName())), "Yes") %>'>
+								<c:when test='<%= StringUtil.equals(ddmFormViewFormInstanceRecordsDisplayContext.getColumnValue(ddmFormFieldsMap.get(ddmFormField.getName()), StringPool.BLANK, ddmFormFieldValuesMap.get(ddmFormField.getName())), LanguageUtil.get(ddmFormViewFormInstanceRecordsDisplayContext.getDefaultLocale(formInstanceRecord), "true")) %>'>
```

Comment:
@mlan0, why did you decide to invoke the method `getDefaultLocale`? Will it work as intended if the user's language is Spanish, for example? Do you remember our discussion about portal languages (instance language, site language and user language)?